app/controllers/api/issues_controller.rb

```diff
@@ -19,14 +19,16 @@ def update_issue
 
     def close_issue
       user, repo, number = params[:user], params[:repo], params[:number]
-      @issue = huboard.board(user, repo).issue(number).close
-      render json: @issue
+      @issue = huboard.board(user, repo).issue(number)
+      @issue.close
+      render json: @issue.activities
```

Comment:
> I decided against creating the object client-side

I agree, I'd like the server to handle it for sure. 

> I suppose the incoming events could also create an in-memory activity object as well... thoughts ?

we could do that but it would be better to standardize on a method of doing so. I have two concerns.

 1. internally generated events could possibly duplicate events from GitHub (much like our troubles with webhook events coming in on closes/reopens done in HuBoard)
 2. code duplication on the client and server around generating activities